tests/src/test/scala/org/apache/openwhisk/core/scheduler/queue/test/MemoryQueueFlowTests.scala

```diff
@@ -1496,6 +1497,7 @@ class MemoryQueueFlowTests
           // queue should not be terminated as there is an activation
           Thread.sleep(gracefulShutdownTimeout.toMillis)
 
+          fsm.underlyingActor.containers = Set(testContainerId)
```

Comment:
adding this line covers case that there is still a container for old queue, but does not cover case that there is no container for old queue

Reply:
Updated accordingly.